app/views/wizards/steps/_membership_terminated_info.html.haml

```diff
@@ -3,5 +3,8 @@
 -#  or later. See the COPYING file at the top-level directory or at
 -#  https://github.com/hitobito/hitobito_sac_cas.
 
-.alert.alert-info
-  = t('.info', date: l(c.model.termination_date))
+- if c.model.termination_date
+  .alert.alert-info
+    = t('.info', date: l(c.model.termination_date))
+- else
+  = "No Termination date"
```

Comment:
There should always be a termination date if this steps gets shown, no?

Reply:
I had an error where this helped to troubleshoot, but will remove now